uwblueprint/childrens-aid-society
# Jackie/delete case action

## Notion ticket link

[Case-Deletion-Action](https://www.notion.so/uwblueprintexecs/Case-Deletion-Action-03490a6c558a4b8daeebf352c3795c60?pvs=4)



## Implementation description
* Backend delete intake route
* Edited models to make delete on cascade work 
* Implemented confirm button functionality on permanent deletion model 
* Created deleteintake method in frontend api client



## Steps to test
1 .Click on "test status modal" 
2. Click on delete
3. Verify that the permanent delete modal works and when you type in confirm delete in textbox
4. Verify api call has been made when you click "confirm delete"



## What should reviewers focus on?
* my changes to the models doesnt have any unintentional effects 


## Checklist
